index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Andika&family=Inter:wght@300;400;500;700&family=Poppins:ital,wght@0,200;0,300;0,500;0,600;0,700;0,800;0,900;1,200&family=Roboto:wght@100;300;400;500;700;900&family=Ubuntu:ital,wght@1,300&display=swap"
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+    </div>
+  </nav>
+  </header>
+  <!--end of nav -->
+  <!--start of  section-->
+  <section class="section-wrapper">
+    <div class="section-header">
+      <h1>I'm Abdullah<br>Glad to see you!</h1>
+    </div>
+    <div class="content-of-section">
+      <p>I'm a software developer!<br> I can help you build a product feature or a website. 
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me.
+    </p>
+    </div>
+    <div class="social-icons">
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium" aria-hidden="true"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist" aria-hidden="true"></i></a></li>
+      </ul>
```

Comment:
- [ ] Icons are looking great! Let's use the color from the template:

Template:
![Captura de pantalla 2022-04-22 a la(s) 11 57 55 p  m](https://user-images.githubusercontent.com/83717285/164878490-32c9707c-98ec-4a3b-b16e-87cb0338a993.png)


Your design:
![Captura de pantalla 2022-04-22 a la(s) 11 58 07 p  m](https://user-images.githubusercontent.com/83717285/164878497-b23689d0-8c56-4536-af05-18707549d71c.png)